### PR TITLE
[Upcoming] Release 1.3.0

### PR DESCRIPTION
**Pull request by skoudoro**
Preparation for 1.3.0 release, targeting Monday, October 26. 

# Desired PR (need review / feedback / update)

There is room for the following PR to be finished and merge

- [x] #2266  (Submitter: @grlee77 )
- [x] #2265   (Submitter: @grlee77  )
- [x] #2264(Submitter: @jhlegarreta  )
- [x] #2263 (Submitter: @skoudoro)
- [x] #2259 (Submitter: @skoudoro )
- [x] #2255 (Submitter: @karanphil)
- [x] #2238 (Submitter: @skoudoro  )
- [x] #2223 (Submitter: @skoudoro )
- [x] #2206 (Submitter: @CHrlS98  )
- [x] #2181 (Submitter: @BramshQamar   )

A lot of PR's, but most of them can be done on time, I think :smile:

Please comment to flag any other issues or PR that should be addressed for this release

# Release checklist
- [x] add release_notes 1.3.0
- [x] Update changelog and api_changes
- [x] Update .mailmap
- [ ] Set release number in info.py and doc/conf.py
- [x] Check the copyright years in doc/conf.py and LICENSE
- [x] Check documentation
- [x] Check Tutorial
- [x] Check unit test and USE-PRE
- [x] Update index.rst
- [x] Update Highligth.rst, old_news.rst, stateofheart.rst
- [x] Update developers.rst list
- [ ] Update website
- [x] Update deprecation cycle

### CODE REVIEW

**Comment by skoudoro** [comment]
Hi @arokem, hi @Garyfallidis,

Can you have a look at this PR and specially [this block](https://github.com/dipy/dipy/pull/2270/files#diff-3161f66a61b9ed68303607f1dbdce1c7bdcc69c42f8ea1c2051a07a579ebcf5dR19-R30)

File: doc/index.rst [hunk 1]
```diff
@@ -16,20 +16,18 @@ perfusion and structural imaging.
...
-- Fixed compatibilities with FURY 0.6+, VTK9+, CVXPY 1.1+.
-- Added multiple tutorials for DIPY command line interfaces.
-- Updated SH basis convention.
-- Improved performance of tissue classification.
-- Fixed a memory overlap bug (multi_median).
-- Large documentation update (typography / references).
-- Closed 256 issues and merged 94 pull requests.
+**DIPY 1.3.0** is now available. New features include:
+
+- Gibbs Ringing speedup increased by 10.
```
Comment (arokem) [inline doc/index.rst:21]:
```suggestion
- Gibbs ringing correction speedup increased by 10x.
```

File: doc/index.rst [hunk 2]
```diff
@@ -16,20 +16,18 @@ perfusion and structural imaging.
...
+- Spherical Harmonics basis definition updated.
```
Comment (arokem) [inline doc/index.rst:22]:
```suggestion
- Spherical harmonics basis definition updated.
```

File: doc/index.rst [hunk 3]
```diff
@@ -16,20 +16,18 @@ perfusion and structural imaging.
...
+- Added SMT2 metrics from mean signal Kurtosis.
```
Comment (arokem) [inline doc/index.rst:23]:
```suggestion
- Added SMT2 metrics from mean signal kurtosis.
```

File: doc/index.rst [hunk 4]
```diff
@@ -16,20 +16,18 @@ perfusion and structural imaging.
...
+- New interface functions to registration module.
+- New Linear transform for registration module
+- New tutorials for DIPY Command Line Interfaces.
```
Comment (arokem) [inline doc/index.rst:26]:
```suggestion
- New tutorials for DIPY command line interfaces.
```

File: doc/index.rst [hunk 5]
```diff
@@ -16,20 +16,18 @@ perfusion and structural imaging.
...
+- Fixed compatibilities with our dependencies.
+- New dependency added: tqdm
```
Comment (arokem) [inline doc/index.rst:28]:
```suggestion
- New progress bar for data downloading, with new dependency added: tqdm
```

**Review by arokem — COMMENTED** [review]
A few small suggestions on the release notes

**Comment by arokem** [comment]
Looks ready to go! @Garyfallidis : do you want to take a look and merge when you think everything is ready for the release?

**Comment by skoudoro** [comment]
Indeed, it is ready to go, @arokem and @Garyfallidis are adressed!

File: Changelog [hunk 6]
```diff
@@ -24,6 +24,20 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 1.3.0 (Tuesday, November 3, 2020)
+
+- Gibbs Ringing correction 10X faster.
```
Comment (arokem) [inline Changelog:29]:
```suggestion
- Gibbs ringing correction 10X faster.
```

File: Changelog [hunk 7]
```diff
@@ -24,6 +24,20 @@ Dipy
...
+- Spherical harmonics basis definitions updated.
+- Added SMT2 metrics from mean signal diffusion kurtosis.
+- New interface functions added to the registration module.
+- New Linear transform added to the registration module.
```
Comment (arokem) [inline Changelog:33]:
```suggestion
- New ``Linear`` transform added to the registration module.
```

File: doc/release_notes/release1.3.rst [hunk 8]
```diff
@@ -0,0 +1,171 @@
+.. _release1.3:
+
+====================================
+ Release notes for DIPY version 1.3
+====================================
+
+GitHub stats for 2020/09/09 - 2020/11/02 (tag: 1.2.0)
+
+These lists are automatically generated, and may be incomplete or contain duplicates.
```
Comment (arokem) [inline doc/release_notes/release1.3.rst:9]:
```suggestion
```

**Review by arokem — COMMENTED** [review]
A couple more small things

**Comment by skoudoro** [comment]
ok, this is ready to go! Anyone for the last look?

**Comment by arokem** [comment]
GGT!

**Comment by Garyfallidis** [comment]
Looks good to me.

**Pull request merged**